pymc/backends/arviz.py

```diff
@@ -63,6 +63,39 @@ def find_observations(model: Optional["Model"]) -> Optional[Dict[str, Var]]:
     return observations
 
 
+def find_constants(model: Optional["Model"]) -> Dict[str, Var]:
+    """If there are constants available, return them as a dictionary."""
+    # The constant data vars must be either pm.Data or TensorConstant or SharedVariable
+    if model is None:
```

Comment:
Same here

Reply:
Hi, I think it better than the previous codes now. Why we did not remove it in previous review [here](https://github.com/pymc-devs/pymc/commit/fe2d101bb27e05b889eafda7e54b07e05250faee#diff-d11fbc930fa0db01efa57955e9481ca2605187656bc12a0326a44d090094698cR47)? But it could be good to have some defensive codes here, since we not 100% sure people always call it with a model. Any way, this is an easy fix, and just a minor thing.

Reply:
Why we did not remove it in previous review here? 

We just missed it then.

> But it could be good to have some defensive codes here, since we not 100% sure people always call it with a model. 

I don't see why you would call this function if not with a model. It just doesn't make sense. And if someone does it that's probably a mistake and the best thing is to let it naturally crash.

Adding special cases like this can make it more difficult to spot errors instead of helping.